.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
Double check the original code, there should be the following dot files which probably got lost as "hidden files" when creating this repo:
* `.eslintrc.js`
* `.gitignore`
* `.nvmrc`
* `.prettierrc.js`